src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Raw.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+
+namespace System.Text.Json
+{
+    public sealed partial class Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the input as JSON content.
+        /// </summary>
+        /// <param name="json">The raw JSON content to write.</param>
+        /// <param name="skipInputValidation">Whether to skip validation of the input JSON content.</param>
+        public void WriteRawValue(string json, bool skipInputValidation = false)
+        {
+            if (json == null)
```

Comment:
I think this is fine, but will pose the question nonetheless, `WriteStringValue(null)` writes the null token, should we do the same here?
https://github.com/dotnet/runtime/blob/3c4d419ffe074ab62ff25a75e4127dbe7c9ee2ab/src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.String.cs#L49-L52

Reply:
I would say probably not, since that would violate the expectation of this only writing raw data.